.coveragerc

```diff
@@ -0,0 +1,32 @@
+# .coveragerc to control coverage.py
+[run]
+branch = True
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+	# Have to re-enable the standard pragma
+	pragma: no cover
+	
+	# Don't complain about missing debug-only code:
+	#def __repr__
+	#if self\.debug
+	
+	# Don't complain if tests don't hit defensive assertion code:
+	#raise AssertionError
+	raise NotImplementedError
+	except ImportError
+	except TypeError
+	except ArchiveConnectionError
+	raise httpexceptions
+	except KeyError
+	except Exception as e:
```

Comment:
I don't think we should exclude all these exceptions and errors.  It's reasonable to exclude `NotImplementedError`, `ImportError`, but really everything else should be tested.  Could you also remove the extra commented out lines you copied?

May be we should completely omit the base storage file and may be other storage files except postgres?  What do people think?